protos/tink/README.md

```diff
@@ -0,0 +1,65 @@
+# How to generate ecdhes_aead protobuf
+
+To execute the proto generation of `ecdhes_aead.proto`, copy this file into
```

Comment:
Seems like we should also add a script and make target for these steps?

Reply:
generating the protobuf at each built is probably overkill as it requires checking out Tink then add and run the new build target mentioned in this readme. 

The go proto is generated and included in this PR to avoid rebuilds. But we can probably automate this if needed later..